pkg/pillar/cmd/zedagent/attesttask.go

```diff
@@ -129,6 +131,29 @@ func (server *VerifierImpl) SendNonceRequest(ctx *zattest.Context) error {
 	return nil
 }
 
+//encodeVersions fetches fetches EVE, UEFI versions
+func encodeVersions(quoteMsg *attest.ZAttestQuote) error {
+	quoteMsg.Versions = make([]*attest.AttestVersionInfo, 0)
+	eveVersion := new(attest.AttestVersionInfo)
+	eveVersion.VersionType = attest.AttestVersionType_ATTEST_VERSION_TYPE_EVE
+	eveRelease, err := ioutil.ReadFile(types.EveVersionFile)
+	if err != nil {
+		return err
+	}
+	eveVersion.Version = string(eveRelease)
+
+	quoteMsg.Versions = append(quoteMsg.Versions, eveVersion)
+
+	uefiVersion := new(attest.AttestVersionInfo)
+	uefiVersion.VersionType = attest.AttestVersionType_ATTEST_VERSION_TYPE_FIRMWARE
+	biosVendor, biosVersion, biosReleaseDate := hardware.GetDeviceBios()
```

Comment:
Does this fail on an ARM device with no BIOS, or does it return empty strings?

Reply:
Tested it on RPi, and verified that empty strings are returned from GetDeviceBios, and hence we omit them while sending to Controller